apps/examples/security_test/security_test_main.c

```diff
@@ -0,0 +1,1623 @@
+										 0x08, 0x01, 0x01, 0x07, 0xa1, 0x44, 0x03, 0x42, 0x00, 0x04, 0x1a, 0xbc, 0xb3, 0x24, 0xa4, 0x1e,
+										 0x89, 0x79, 0x6e, 0xbe, 0x75, 0x7f, 0x78, 0xa7, 0x32, 0x21, 0xae, 0x7b, 0xb5, 0xe4, 0xa4, 0x4d,
+										 0x07, 0x44, 0x0e, 0x07, 0x5a, 0x77, 0x3d, 0xb3, 0xd8, 0xfc, 0x65, 0x02, 0xfc, 0xd7, 0x1a, 0x3c,
+										 0xcb, 0x9a, 0x4e, 0x34, 0xde, 0x32, 0xe8, 0x3e, 0x2e, 0xda, 0x09, 0xe0, 0x5f, 0x24, 0x7b, 0x86,
+										 0x83, 0x08, 0xc3, 0xf6, 0x7f, 0xe3, 0x81, 0xbb, 0xda, 0x62
+									   };
+
+unsigned char test_ecdsa_dev_nist[223] = { 0x30, 0x81, 0xdc, 0x02, 0x01, 0x01, 0x04, 0x42, 0x00, 0x94, 0x4f, 0x20, 0xa2, 0x5b, 0x75, 0xcb,
+										   0x2d, 0xc6, 0x17, 0xd1, 0xd9, 0x04, 0xb5, 0x19, 0x4b, 0xfd, 0x80, 0x8f, 0xd8, 0x6f, 0xc0, 0x23,
+										   0x2f, 0x23, 0x90, 0xe8, 0x29, 0xa9, 0x9e, 0x28, 0x0b, 0x8d, 0xa8, 0xf2, 0xb5, 0x4a, 0x7d, 0xc8,
+										   0x09, 0x79, 0xb7, 0xb4, 0xc4, 0x6e, 0xb4, 0x95, 0x35, 0xbe, 0xcd, 0x19, 0x96, 0x9b, 0x3c, 0xae,
+										   0x50, 0x2c, 0x98, 0x77, 0x47, 0x0c, 0x7f, 0x87, 0x3b, 0xbd, 0xa0, 0x07, 0x06, 0x05, 0x2b, 0x81,
+										   0x04, 0x00, 0x23, 0xa1, 0x81, 0x89, 0x03, 0x81, 0x86, 0x00, 0x04, 0x00, 0xa7, 0xe3, 0x75, 0x81,
+										   0x42, 0x82, 0x15, 0x7c, 0x36, 0xff, 0x46, 0x43, 0x78, 0x9f, 0x94, 0x99, 0x5c, 0xba, 0x59, 0x90,
+										   0xdc, 0x2a, 0xed, 0x90, 0xbb, 0x87, 0x07, 0x99, 0x69, 0x17, 0xa5, 0x1d, 0x38, 0x71, 0xc9, 0x91,
+										   0x0e, 0x9c, 0xa6, 0x35, 0x30, 0x91, 0x1e, 0x98, 0x07, 0x5c, 0xfd, 0xe2, 0x8c, 0x44, 0x16, 0xc3,
+										   0xc4, 0x2b, 0x03, 0xf0, 0x3b, 0xd7, 0xdc, 0x26, 0xcb, 0x82, 0xe5, 0xcd, 0x47, 0x01, 0xc3, 0x01,
+										   0xa9, 0xd6, 0xe3, 0xe3, 0x65, 0x4a, 0x8d, 0x23, 0xff, 0xb4, 0xaa, 0xa6, 0x32, 0x30, 0x63, 0x8c,
+										   0x31, 0x1a, 0x83, 0xa7, 0xd6, 0xd1, 0xf1, 0xd2, 0x50, 0xbf, 0xd5, 0x51, 0x3b, 0x74, 0x13, 0x71,
+										   0x79, 0x1b, 0x55, 0x34, 0x2b, 0xae, 0x98, 0x40, 0x32, 0xcc, 0xf4, 0xae, 0xc5, 0xe3, 0xd3, 0x03,
+										   0xcf, 0x8f, 0x36, 0xeb, 0x2c, 0xc8, 0x0a, 0xc7, 0xad, 0x88, 0x24, 0x9f, 0x00, 0xd4, 0x18
+										 };
+
+unsigned int TC_see_read_write_secure_storage(void)
+{
+	int j = 0;
+	int ret = 0;
+	unsigned int test_result = 0;
+	unsigned char input[300];
+	unsigned int inlen[3] = { 18, SEE_MAX_DATA_SIZE, SEE_MAX_DATA_SIZE + 1 };
+	unsigned char output[SEE_MAX_DATA_SIZE];
+
+	printf("--------------------------------------------\n");
+	printf("  TESTCASE : see_read/write_secure_storage\n");
+	printf("--------------------------------------------\n");
+
+	memset(input, 1, 300);
+
+	for (j = 0; j < SSTORAGE_DATA_SLOT_INDEX_MAX; j++) {
+		see_selfprintf("[%d] ", j);
+
+		/* positive case 1 */
+		memset(output, 0, SEE_MAX_DATA_SIZE);
+		ret = see_write_secure_storage(input, inlen[0], j);
+		if (ret != SEE_OK) {
+			see_selfprintf("SS write fail\n");
+			break;
+		}
+
+		ret = see_read_secure_storage(output, &inlen[0], j);
+		if (ret != SEE_OK) {
```

Comment:
In my opinion, it is also necessary to compare write -> read len.
if (ret != SEE_OK || write_len != read_len)